scripts/setup

```diff
@@ -69,48 +68,51 @@ pipenv run invenio roles add admin@rero.ch admins
 pipenv run invenio roles add admin@rero.ch superusers
 
 
-# create the organisations with libraries and locations
-pipenv run invenio fixtures importorganisations ${data_path}/organisations-libraries-locations.json -v
+pipenv run invenio records create --pid-minter organisation_id data/organisations.json
+pipenv run invenio records create --pid-minter library_id data/libraries.json
+pipenv run invenio records create --pid-minter location_id data/locations.json
+pipenv run invenio records create --pid-minter item_type_id data/item_types.json
+pipenv run invenio records create --pid-minter patron_type_id data/patron_types.json
+pipenv run invenio records create --pid-minter circ_policy_id data/circulation_policies.json
 
-# create patron types
-# NEEDS organisations !!!
-pipenv run invenio fixtures importpatronstypes ${data_path}/patron_types.json
+pipenv run invenio index reindex -t lib --yes-i-know
+pipenv run invenio index reindex -t loc --yes-i-know
+pipenv run invenio index reindex -t itty --yes-i-know
+pipenv run invenio index reindex -t ptty --yes-i-know
+pipenv run invenio index reindex -t cipo --yes-i-know
+pipenv run invenio index run --raise-on-error
 
-# create item types
-# NEEDS organisations !!!
-pipenv run invenio fixtures importitemstypes ${data_path}/item_types.json
+pipenv run invenio fixtures import_users ${data_path}/users.json -v
 
-# create the patron records
-# NEEDS patron type !!!
-pipenv run invenio fixtures importusers ${data_path}/users.json -v
 
-# create circulation policies
-# NEEDS organisation !!!
-pipenv run invenio fixtures importcircpolicies ${data_path}/circulation_policies.json
-
-# API Mef configurations and harvest
+## API Mef configurations and harvest
 pipenv run invenio apiharvester sources ${data_path}/apisources.yml
 if [ "$1" = "--deployment" ]
 then
-    # start api harvesting asynchrone: beats must be running
-    pipenv run invenio apiharvester harvest -n mef -k
+   # start api harvesting asynchrone: beats must be running
+   pipenv run invenio apiharvester harvest -n mef -k
 else
     pipenv run invenio apiharvester harvest -n mef -m 1000
 fi
 
-# create the bib records
-pipenv run dojson -i ${data_path}/10k.json reverse schema http://ils.test.rero.ch/schema/documents/document-v0.0.1.json | pipenv run invenio records create --pid-minter document_id
-pipenv run invenio index reindex --yes-i-know --pid-type doc
-pipenv run invenio index run -c 4
+# # create the bib records
+pipenv run dojson -i data/10k.json schema 'http://ils.rero.ch/schema/documents/document-v0.0.1.json' | pipenv run invenio records create --pid-minter document_id
+pipenv run invenio index reindex -t doc --yes-i-know
+pipenv run invenio index run -c 4 --raise-on-error
+
+# # create items
+# # NEEDS item type !!!
+# to generate data file
+# pipenv run invenio fixtures createitems -c 1000 -i 5 data/items.json
```

Comment:
This command should be launched manually when one needs to generate data file?
We should have also an item data file for deployement (with items for all bib rec excepts ebooks).

Reply:
Why do we needs different items for dev and deploy?

Reply:
@jma It could change now, with your new setup and `data/items.json` file. But once upon a time, it was very very very very long to create 10k or more items at each setup. So someone added an option to create only 1k items for dev. And we began deploying only 1k items on deploy, which isn't a good idea.

Reply:
@all The secret to instantly loading anything is the implementation of the separate tables (see how fast we load the MEF data now). so keep calm and push the the Merge button :-)

Reply:
@BadrAly Yeah. I just want to be sure to have a `data/items.json` with **more than 1k items** for deployement purpose. We just have to create 'em once. That's all.

Reply:
@iGormilhit probably we can do better: having a `data/items.json` with 10k or more and load only the first 1k. This can be done using the head dojson filter. Ok?

Reply:
@jma :+1: